.gitignore

```diff
@@ -15,4 +15,6 @@ env/
 venv/
 ENV/
 env.bak/
-venv.bak/
\ No newline at end of file
+venv.bak/
+data/settings.json
+data/settings.json
```

Comment:
You put data/settings.json twice.

Reply:
Thank you, that has been corrected.  (If you don't want data/settings.json in .gitignore, I can remove that as it really didn't belong in this commit anyway.  I figured it would be good to prevent usernames/passwords, encrypted as they may be, from being committed.)